packages-new/components/accordion/src/components/osds-accordion/osds-accordion.e2e.screenshot.ts

```diff
@@ -1,16 +1,18 @@
 import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
-import { OdsAccordionAttributes, OdsAccordionSizeList, OdsComponentAttributes2StringAttributes, odsAccordionDefaultAttributes } from "@ovhcloud/ods-core";
-import { OdsCreateAttributes, OdsStringAttributes2Str, odsAccordionBaseAttributes } from "@ovhcloud/ods-testing";
 
 import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import { DEFAULT_ATTRIBUTE } from "./constants/default-attributes";
+import { OdsAccordionAttributes } from "./interfaces/attributes";
+import { OdsComponentAttributes2StringAttributes } from "@ovhcloud/ods-common-core";
+import { OdsStringAttributes2Str } from "@ovhcloud/ods-common-testing";
```

Comment:
please use `odsStringAttributes2Str` instead
(sadly the bare copy of the libs have also kept things that will be removed soon)